modules/01-kafka-on-openshift.md

```diff
@@ -98,4 +98,75 @@ The `KafkaUser` resources are described in the `kafka/kafka-users.yaml` file.
 
 ```shell
 oc apply -f kafka/kafka-users.yaml
+```
+
+## Deploying resources for visualizing metrics
+
+To visualize metrics from Kafka, Kafka Connect and Zookeeper containers we need to deploy visualization chain. The first segment is Prometheus Operator by CoreOS.
+This operator allows us to easily deploy another segment - Prometheus server. With Prometheus server deployed we will need the last segment - Grafana.
+Grafana is a graphical tool which displays selected metrics very easily.
+
+### Deploying Prometheus Operator
+To deploy Prometheus Operator we can use [bundle file](https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml) provided by CoreOS. 
+
+The file contains all the neccessary resources which are needed to deploy this operator:
+- ClusterRole - permissions to manipulate with resources
+- ClusterRoleBinding - binding between CLusterRole and ServiceAccount
+- Deployment - specification of the Prometheus Operator deployment
+- ServiceAccount
+- Service
+
+This file is generated so you should modify it. You should remove all the `namespace: default` values except of one in `ClusterRoleBinding`. Here you should put your namespace.
+When deploying on OpenShift you should also remove `spec.securityContext` from the `Deployment`.
+
+After these modifications you can use
+```shell
+oc apply -f bundle.yaml
+```
+
+### Deploying Prometheus and Alertmanager
+When the Prometheus Operator is deployed, you can send a `Prometheus` and `Alertmanager` resource to the OpenShift server API and Prometheus Operator should deploy desired number of Prometheus and Alertmanager servers.
+
+Before we do that, we need to prepare a `Secret`s resources which are used by servers. These `Secret`s contain additional configuration for Prometheus and Alertmanager servers.
+
+```shell
+TBD: adding entire prometheus dir to this repo?
```

Comment:
we need all the files needed for the deployment or if no changes we can use the files in the upstream repo.